app/src/main/java/org/jellyfin/mobile/webapp/WebappFunctionChannel.kt

```diff
@@ -14,10 +14,10 @@ class WebappFunctionChannel {
     fun call(action: String) = internalChannel.offer(action)
 
     // Web component helpers
-    private fun callWebComponent(component: String, cmd: String) = call("require(['$component'], function($component){$component.$cmd;});")
-    fun triggerInputManagerAction(action: String) = callWebComponent("inputManager", "trigger('$action')")
-    fun seekTo(pos: Long) = callWebComponent("inputManager", "trigger('seek', $pos)")
-    fun setVolume(volume: Int) = callWebComponent("playbackManager", "sendCommand({Name: 'SetVolume', Arguments: {Volume: $volume}})")
+    fun callPlaybackManager(action: String) = call("window.NavigationHelper.playbackManager.$action();")
```

Comment:
```suggestion
    fun callPlaybackManager(action: String) = call("window['NavigationHelper'].playbackManager.$action();")
```

Reply:
Some functions calling this function already invoke the function (like setVolume and seekTo below) which will make this mess: `window.NavigationHelper.playbackManager.setVolume(...)();`